www/styles/Notification.module.css

```diff
@@ -0,0 +1,76 @@
+.container {
```

Comment:
you will need to add `z-index` here in order to have the the notification bar appear above the page under any circumstance.